File: web/src/app/Models/Dtos/Test/candidate-assessment-dto.ts
import { DifficultyLevel } from '../../Enums/DifficultyLevel';
import { CandidateQuestionDto } from './candidate-question-dto';

export interface CandidateAssessmentDto {
  id: string;
  title: string;
  difficultyLevel: DifficultyLevel;
  questions: CandidateQuestionDto[];
}

File: backend/SkillProof/SkillProof.Entities/Dtos/Tests/CandidateAssessmentDto.cs
using SkillProof.Entities.Enums;

namespace SkillProof.Entities.Dtos.Tests;

public class CandidateAssessmentDto
{
    public string Id { get; set; } = string.Empty;
    public string Title { get; set; } = string.Empty;
    public DifficultyLevel DifficultyLevel { get; set; }
    public List<CandidateQuestionDto> Questions { get; set; } = new();
}
